### PR TITLE
New outfit tutorial

### PR DESCRIPTION
**Pull request by JojjeE**
Covers:
- Charset
- Outfit
- Globaldata
- Blobs
- Localization
- Peacock (Repository and unlockables)
- Chongqing rain logic

I think this is our longest yet clocking in at a little under 6800 words. Please give feedback if anything looks weird or I haven't explained something well enough. Brain mushy. Thanks.

### CODE REVIEW

File: docs/modding/suitmodding/newoutfit.md [hunk 1]
```diff
@@ -0,0 +1,648 @@
...
+-   Footstep sounds
+-   Voice modulation for masks
+-   Any sound effects that should be used when moving ("foley")
+-   Professions, which control enforcers and trespassing areas for the player
+-   Voice variations for NPCs
+-   Outfit AI Category, which mainly dictates how NPCs address the player
+-   Which character set to use
+-   Name and description, which is for the disguise bags in the level
+
+There is also a brick called `globaldata`. This is a brick that the game always loads, and it includes important basic gameplay elements. This includes 47's starter outfits, any outfit you can go into a level with.
```
Comment (Notexe) [inline docs/modding/suitmodding/newoutfit.md]: Maybe change this to "This includes 47's starter outfits and any outfit you can go into a level with." just to make it read a bit better.

File: docs/modding/suitmodding/newoutfit.md [hunk 2]
```diff
@@ -0,0 +1,648 @@
...
+Save your manifest. Why don't we try deploying the mod in SMF and booting up the game? Give it a try, and if you've done everything right so far, your results should speak for themselves!
+
+![The new inventory image, name and description.](/img/suitmodding/newoutfit/finished_01.jpg)
+![Wearing the finished outfit.](/img/suitmodding/newoutfit/finished_02.jpg)
+
+For completion's sake, in your manifest, you should probably not stop with just English localization. If you have the means, you should translate to as many languages as you can. Intellisense will tell you which languages are valid. The more translations you provide, the bigger the potential audience of your mod, which is always cool.
+
+## Extra Credit: Peacock Addon
+
+Your mod is finished now, if your goal was just to replace a suit with your own. But we can go the extra mile too, and add our suit to the game without replacing any others. This will only work if the mod user is playing on Peacock, however. Doing this will require modifying the repository and unlockables which won't do anything if we play on IOI's servers, as we mentioned earlier.
```
Comment (Notexe) [inline docs/modding/suitmodding/newoutfit.md]: Might be worth mentioning that this also works in offline mode

**Edit:** nvm that requires the starter package thing I think

**Review by Notexe — COMMENTED** [review]
From what I can tell everything seems to be good. I'm not really too sure if it's worth adding information about patching the starter package thing for offline mode support or not.

File: docs/modding/suitmodding/newoutfit.md (hunk 2 above)
Comment (JojjeE) [reply docs/modding/suitmodding/newoutfit.md]: Ah I didn't consider offline mode. Yeah that would be the purpose of the starter package patch. I already wrote it so I may as well reincorporate it slightly reworded.

**Comment by Notexe** [comment]
Seems good to me, will merge it

**Pull request merged**